### PR TITLE
Feature/core 1176 streamline ui

### PR DESCRIPTION
**Pull request by karladler**

### CODE REVIEW

File: core/evancore.vue.libs/src/components/nav-list/nav-list.scss [hunk 1]
```diff
@@ -106,5 +110,10 @@
         }
       }
     }
+
+    button:first-of-type {
+      margin-top: 0;
+      box-shadow: 230px 0px 3px 3px cssVar('bg-level-3'), 10px 3px 3px -3px cssVar('primary');
+    }
```
Comment (datrinh) [inline core/evancore.vue.libs/src/components/nav-list/nav-list.scss:117]: Do I understand it correctly, that you omit the shadow on top for the first entry? If so, that's not what the figma shows?

Comment (karladler) [reply core/evancore.vue.libs/src/components/nav-list/nav-list.scss:117]: That's true, but necessary for the third level menu. Looks okay for me, but we can discuss it.

File: core/evancore.vue.libs/src/components/dapp-wrapper/mnemonic-export/mnemonic-export.vue [hunk 2]
```diff
@@ -43,15 +43,20 @@
...
+              />
+              <p
+                class="hyphen-auto pre-wrap text-justify"
+                v-html="$t('_evan.mnemonic-export.description')"
+              />
+              <p
+                class="hyphen-auto pre-wrap text-justify bold no-print"
+                v-html="$t('_evan.mnemonic-export.print-or-store')"
+              />
+              <!-- eslint-enable vue/no-v-html -->
```
[the remark this answers is not in the document]
Comment (karladler) [reply core/evancore.vue.libs/src/components/dapp-wrapper/mnemonic-export/mnemonic-export.vue]: not using the directive here would still add one space, but I found another solution to fix it.

File: core/evancore.vue.libs/src/components/nav-list/nav-list.scss (hunk 1 above)
Comment (datrinh) [reply core/evancore.vue.libs/src/components/nav-list/nav-list.scss:117]: Would maybe better to add a class for 3rd level submenu instead and only apply first-of-type there?

**Pull request merged**